src/registry_schemas/example_data/schema_data.py

```diff
@@ -1226,7 +1226,87 @@
 
 RESTORATION = {
     'date': '2021-01-30',
-    'type': 'fullRestoration'
+    'type': 'fullRestoration',
+    'nameRequest': {
+        'legalType': 'BC',
+    },
+    'nameTranslations': [
+        {'name': 'ABCD Ltd.'}
+    ],
+    'approvalType': 'courtOrder',
+    'courtOrder': COURT_ORDER,
```

Comment:
Court order is normally an object, I think.

I know this is a different use of Court Order number (aka file number), but ...?

Reply:
it is an object

Reply:
Oh, that's what that means 😊